src/current/_includes/releases/v24.1/v24.1.0-rc.1.md

```diff
@@ -0,0 +1,57 @@
+## v24.1.0-rc.1
+
+Release Date: May 8, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-0-rc-1-sql-language-changes">SQL language changes</h3>
+
+- Added a new session setting `optimizer_use_improved_multi_column_selectivity_estimate`, which if enabled, causes the [optimizer]({% link v24.1/cost-based-optimizer.md %}) to use an improved selectivity estimate for multi-column predicates. This setting will default to `true` on v24.2 and later, and `false` on earlier versions. [#123106][#123106]
+
+<h3 id="v24-1-0-rc-1-operational-changes">Operational changes</h3>
+
+- Added two new metrics: `range.snapshots.upreplication.rcvd-bytes` counts the number of Raft recovery snapshot bytes received, and `range.snapshots.upreplication.sent-bytes` counts the number of Raft recovery snapshot bytes sent. Also updated `range.snapshots.recovery.rcvd-bytes` and `range.snapshots.recovery.sent-bytes` to only include Raft snapshots. A new line was added to the [**Snapshot Data Received**]({% link v24.1/ui-replication-dashboard.md %}#snapshot-data-received) graph. [#123055][#123055]
+
+<h3 id="v24-1-0-rc-1-db-console-changes">DB Console changes</h3>
+
+- Added a **Replication Lag** graph, which measures replication lag between primary and standby clusters for [physical cluster replication]({% link v24.1/physical-cluster-replication-overview.md %}), to the [**Physical Cluster Replication**]({% link v24.1/physical-cluster-replication-monitoring.md %}) dashboard. [#123285][#123285]
+
+<h3 id="v24-1-0-rc-1-bug-fixes">Bug fixes</h3>
+
+- The [**Tables**]({% link v24.1/ui-databases-page.md %}#tables-view) and [**Table details**]({% link v24.1/ui-databases-page.md %}#table-details) pages in the DB Console now display the correct **Table Stats Last Updated** value. [#122816][#122816]
+- Fixed a bug in the DB Console's [**Custom Chart**]({% link v24.1/ui-custom-chart-debug-page.md %}) tool where store-level metrics were not being displayed properly. Previously, if a store-level metric was selected to be displayed at the store-level on a multi-store node, only data for the 1st store ID associated with that node would be displayed.  This patch ensures that data is displayed for all stores present on a node. Additionally, it updates the behavior to show a single time series for each store, as opposed to aggregating (e.g., `SUM`) all stores across the node. This allows finer-grained observability into store-level metrics. [#122705][#122705]
```

Comment:
```suggestion
- Fixed a bug in the DB Console's [**Custom Chart**]({% link v24.1/ui-custom-chart-debug-page.md %}) tool where store-level metrics were displayed only for the first store ID associated with the node. Now a single time series is shown for each store, rather than an aggregated value for all of the node's stores. This allows finer-grained monitoring of store-level metrics. [#122705][#122705]
```

Reply:
Edited your suggestion to add `data is displayed for all stores present on a node` which was also a key change here.